assets/app/view/game/company.rb

```diff
@@ -109,6 +113,14 @@ def render
             children << render_bidders
           end
 
+          if @company.discount != 0
```

Comment:
if company.discount.positive? or unless company.discount.zero?

Reply:
Unless company.discount.zero. making it positive won't help fix the 1846 issue.

Reply:
The 1846 minors can be discounted too. I'm not sure how it's implemented on this site and it may not even be correct if the implementation overloads discount and debt, since I believe the discount is supposed to apply to the buying price rather than the debt.

Reply:
(Confirmed: https://boardgamegeek.com/thread/1880571/article/27393557#27393557)